Parsers/SysmonParser.txt

```diff
@@ -1,12 +1,16 @@
 // KQL Sysmon Event Parser
-// Date: May 8, 2019
+// Date: June 4, 2019
 // Notes:
 // 1. This parser works against the current version of events from Sysmon, it may need updates if Sysmon is updated.
```

Comment:
Lets add in the version of Sysmon.exe this came from so it is more clear.  This came from version 9.1.0.0

Reply:
sure, added version and xml config reference.